MusicHub/tracks/serializers.py

```diff
@@ -0,0 +1,50 @@
+from rest_framework import serializers
+from rest_framework.serializers import (
+    ModelSerializer,
+    ValidationError,
+)
+from tinytag import TinyTag
+
+from MusicHub.main.utils import format_to_minutes
+from MusicHub.tracks.models import Track
+
+
+class CreateTrackSerializer(ModelSerializer):
+    class Meta:
+        model = Track
+        fields = ["filename", "track", "public"]
+
+    def validate_track(self, value):
+        if value.size > 30000000:
```

Comment:
Fisrtly let's get rid of magic number. Than let's use format like `30 * 1024 * 1024` for the const `MAX_TRACK_SIZE_IN_MB`